specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/examples/changeTenantPolicy.json

```diff
@@ -0,0 +1,30 @@
+{
+  "parameters": {
+    "api-version": "2019-10-01-preview",
+    "body": {
+      "blockSubscriptionsLeavingTenant": true,
+      "blockSubscriptionsIntoTenant": true,
+      "exemptedPrincipals": [
+        "e879cf0f-2b4d-5431-109a-f72fc9868693",
+        "9792da87-c97b-410d-a97d-27021ba09ce6"
+      ]
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "291bba3f-e0a5-47bc-a099-3bdcb2a50a05",
```

Comment:
>291bba3f-e0a5-47bc-a099-3bdcb2a50a05 [](start = 15, length = 36)

Given your current path, the `id` should be `/providers/Microsoft.Subscription/policies/default`. If you are intending to support more than one policy, then the paths would have to be adjusted to support it.